charts/hivemq-platform/values.yaml

```diff
@@ -88,17 +88,41 @@ restApi:
   # Enables or disables authentication and authorization.
   authEnabled: false
 
-# Configures the HiveMQ License information.
+# Configures the HiveMQ Platform license information.
 license:
   # Creates a default HiveMQ Platform license as a Kubernetes Secret.
   create: false
-  # The name of a secret with the license information.
+  # The name of a Kubernetes Secret with the HiveMQ Platform license information.
   # To create a new secret with a license, set create=true. Otherwise, false to reuse an existing one.
   name: ""
-  # Inlines the License information as an encoded 64-bytes string.
+  # Inlines the HiveMQ broker license information as an encoded 64-bytes string.
   data: ""
-  # Overrides the License information via file using --set-file license.lic.
+  # Overrides the HiveMQ broker license information via file using --set-file license.lic.
   overrideLicense: ""
+  # Additional HiveMQ broker licenses to include as .lic files of the same Kubernetes Secret license.
+  additionalBroker: {}
+    ## Defines the name of the HiveMQ broker license in the Kubernetes secret (the .lic suffix will be added automatically).
+    #<broker-license-name>:
```

Comment:
```suggestion
    #license-name:
```

Reply:
Not so sure here regarding the removal of the minor and greater than symbols.... In all of our examples we have commented out in the values.yaml file, these are existing values defined in the JSON schema, so customer can only use them with that name. 
In this particular case though, it's different cause there is no `license-name` value available as such as the customer can choose whatever name they like. So I think the most conventional way of defining this is by using `<>` or something to clearly denote that this can be any value.
For now, I will remove the `broker` prefix. 
WDYT?